demos/simple/argv/src/main.rs

```diff
@@ -0,0 +1,25 @@
+#[cfg(not(verify))]
+use proptest::prelude::*;
+#[cfg(verify)]
+use propverify::prelude::*;
+
+use std::env;
+
+/// Simple test to confirm that it is possible to pass arguments
+/// to Rust functions when verifying them.
+///
+/// Should pass:
+///    `cargo verify . foo`
```

Comment:
'cargo test' passes all the arguments after '--' to the test. Can you make 'cargo verify' behave the same?